docs/release_notes/v1.0.0-rc.3.md

```diff
@@ -0,0 +1,512 @@
+
+- **FIXED** Kubernetes get secret to fallback to k8s namespace. [563](https://github.com/dapr/components-contrib/issues/563)
+
+- **RESOLVED** Add concurrency to RabbitMQ [574](https://github.com/dapr/components-contrib/issues/574)
+
+- **RESOLVED** Exclusive Queues for RabbitMQ [577](https://github.com/dapr/components-contrib/issues/577)
+
+- **RESOLVED** Added Exclusive Flag to RabbitMQ queues [580](https://github.com/dapr/components-contrib/pull/580)
+
+- **RESOLVED** Support priority queue for RabbitMQ Bind/Pubsub Components [582](https://github.com/dapr/components-contrib/issues/582)
+
+- **RESOLVED** App able to subscribe to one GCP Pub/Sub topic only [583](https://github.com/dapr/components-contrib/issues/583)
+
+- **RESOLVED** Shared Subscription in MQTT [589](https://github.com/dapr/components-contrib/issues/589)
+
+- **RESOLVED** SecretStore BulkGetSecret method flattens all of secrets into a single map [590](https://github.com/dapr/components-contrib/issues/590)
+
+- **RESOLVED** Update sample links [611](https://github.com/dapr/components-contrib/pull/611)
+
+- **RESOLVED** Need a MySQL binding support similar with PostgreSQL [616](https://github.com/dapr/components-contrib/issues/616)
+
+- **RESOLVED** Fix get bulk secret for hashicorp vault. [632](https://github.com/dapr/components-contrib/pull/632)
+
+- **RESOLVED** Fix service bus new topic creation [636](https://github.com/dapr/components-contrib/pull/636)
+
+### Dashboard
+
+- **FIXED** display of full cmd and registered actors [78](https://github.com/dapr/dashboard/issues/78)
+
+- **RESOLVED** Numerous improvements and issues on the Dashboard [93](https://github.com/dapr/dashboard/issues/93)
+
+- **RESOLVED** Improve log page for K8s [104](https://github.com/dapr/dashboard/issues/104)
+
+- **ENABLED** Dashboard to run unprivileged. [119](https://github.com/dapr/dashboard/issues/119)
+
+- **FIXED** reported vulnerabilities by npm. [120](https://github.com/dapr/dashboard/issues/120)
+
+### .NET SDK
+
+The .NET SDK NuGet packages thatcorresponds with RC3 of the Dapr runtime use the version `1.0.0-rc04`.
+
+The .NET SDK has several major feature additions and impactful changes in this release. We recommend every user read this [migration guide](https://github.com/dapr/dotnet-sdk/issues/569) as a supplement to the release notes.
+
+- **RESOLVED** Remove Query all statenames from IActorStateManager because it does not behave as expected [53](https://github.com/dapr/dotnet-sdk/issues/53)
+
+- **FIXED** Changed signatures to use IReadOnlyDictionary<TKey, TValue> over Dictionary<TKey, TValue> in public APIs [395](https://github.com/dapr/dotnet-sdk/issues/395)
+
+- **FIXED** Actor Runtime should not report Exception text in HTTP responses [410](https://github.com/dapr/dotnet-sdk/issues/410)
+
+- **Add** support for metada in pub/sbu [425](https://github.com/dapr/dotnet-sdk/issues/425)
```

Comment:
```suggestion
- **Add** support for metadata in pub/sub [425](https://github.com/dapr/dotnet-sdk/issues/425)
```

Reply:
The source of this file is hosted someplace else and synced here. I fixed the source based on your suggestion. Thanks!